frontend/src/components/Restaurants/index.jsx

```diff
@@ -0,0 +1,45 @@
+import { PropTypes } from 'prop-types';
+import { Fragment } from 'react';
+
+const propTypes = {
+    restaurants: PropTypes.array.isRequired,
+    error: PropTypes.string.isRequired
+}
+
+const Restaurants = ({ restaurants, error }) => {
+
+    const addressDisplay = (address) => {
+        return (
+            <Fragment>
+                <p>Street Address: {address.streetNumberAndName}, City: {address.city}, State: {address.stateAbbr}, Zip: {address.zipCode}</p>
+            </Fragment>
+        );
+    }
+
+    const restaurantsDisplay = (restaurants) => {
+        return restaurants.map((restaurant, i) => (
+            <Fragment key = {i}>
+                <p><b>{restaurant.name}</b> </p>
+                <p>{addressDisplay(restaurant.address)}</p>
+            </Fragment>
+        ));
+    }
+
+    if(!restaurants) {
+        return <p>{error}</p>;
+    }
+    else if (restaurants.length === 0) {
+        return <p>No restaurants exist yet</p>
```

Comment:
This error message appears every time after the page is loaded and before the state is populated. In my opinion, it would be better to show a loader component here. And if we receive an error from the backend, then it will update to show the error which we have on line 29.

Reply:
Did not know that thanks for the info!

Reply:
Added in [9b99bcd](https://github.com/NickPriv/FryRank/pull/17/commits/9b99bcd1c8795461a981ef323ec657bb2d09b4a1).